src/components/Oscillators.js

```diff
@@ -86,14 +86,14 @@ export default function Oscillators({
             onClick={handleChangeOsc2Type}
           >
             <Saw />
-          </button>{" "}
+          </button>
           <button
             id="square"
             className={oscillator2Id === "square" ? "Change-Active" : "Change"}
             onClick={handleChangeOsc2Type}
           >
             <Square />
-          </button>{" "}
+          </button>
           <button
             id="sine"
             className={oscillator2Id === "sine" ? "Change-Active" : "Change"}
```

Comment:
removing some {" "} on the fly. I saw them by accident and removed them without thinking too much of the branch. I won't do this again. promised.